test/e2e/upgradetest/framework/framework.go

```diff
@@ -97,10 +97,7 @@ func (f *Framework) CreateOperator() error {
 }
 
 func (f *Framework) DeleteOperator() error {
-	foreground := metav1.DeletePropagationForeground
-	return f.KubeCli.AppsV1beta1().Deployments(f.KubeNS).Delete("etcd-operator", &metav1.DeleteOptions{
-		PropagationPolicy: &foreground,
-	})
+	return f.KubeCli.AppsV1beta1().Deployments(f.KubeNS).Delete("etcd-operator", metav1.NewDeleteOptions(0))
```

Comment:
Is `metav1.NewDeleteOptions(0)` going to work? I think this is not going to delete its children, i.e. RS and pods. Can you verify that?